artman/tasks/publish/sample_app.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, unicode_literals
+import functools
+import os
+import uuid
+
+import github3
+
+import six
+
+from artman.tasks import task_base
+from artman.utils.logger import logger
+
+class _JavaSampleTask(task_base.TaskBase):
+    def execute(self, gapic_code_dir=None, grpc_code_dir=None, proto_code_dir=None):
+        
+        if grpc_code_dir:
```

Comment:
If you're going to raise an exception without `gapic_code_dir` or `proto_code_dir` anyway, why make them optional arguments at all?